requirements.txt

```diff
@@ -1,15 +1,562 @@
+Quamash==0.6.1
+redis==3.5.3
+regex==2021.11.10
+repoze.lru==0.7
+requests==2.28.1
+requests-file==1.5.1
+requests-ntlm==1.1.0
+requests-oauthlib==1.3.1
+requests-toolbelt==0.9.1
+responses==0.18.0
+retrying==1.3.3
+rfc3339-validator==0.1.4
+rfc3986-validator==0.1.1
+rich==12.4.4
+Routes==2.5.1
+rq==1.10.1
+rsa==4.8
+ruamel.yaml==0.17.16
+ruamel.yaml.clib==0.2.6
+rule-engine==1.1.0
+Rx==3.2.0
+scapy==2.4.4
+scikit-learn==1.2.0
+scipy==1.9.3
+seaborn==0.11.2
+SecretStorage==3.3.2
+secure==0.3.0
+selenium==4.0.0a1
+Send2Trash==1.8.0
+service-identity==18.1.0
+setproctitle==1.2.2
+sgmllib3k==1.0.0
+shodan==1.27.0
+simplejson==3.17.6
+simplekv==0.13.0
+six==1.16.0
+slowapi==0.1.4
+smmap==5.0.0
+smoke-zephyr==2.0.1
+sniffio==1.2.0
+sortedcontainers==2.4.0
+soupsieve==2.3.2
+speaklater==1.4
+speedtest-cli==2.1.3
+spyse-python==2.2.3
+SQLAlchemy==1.4.31
+sqlalchemy-schemadisplay==1.3
+SQLAlchemy-Utc==0.11.0
 sqlparse==0.4.3
+sslyze==5.0.5
+stack-data==0.6.2
+starlette==0.18.0
+statsmodels==0.13.5
+stone==3.3.1
+sympy==1.10.1
+syslog-rfc5424-formatter==1.2.2
+systemd-python==234
+tables==3.7.0
+tabulate==0.8.9
+tangled-up-in-unicode==0.2.0
+Tempita==0.5.2
+tempora==5.0.1
+tensorboard==2.9.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-wit==1.8.1
+tensorflow==2.9.1
+tensorflow-estimator==2.9.0
+tensorflow-io-gcs-filesystem==0.26.0
+termcolor==1.1.0
+terminado==0.17.1
+terminaltables==3.1.0
+texttable==1.6.4
+theHarvester==4.0.3
+threadpoolctl==3.1.0
+tinycss2==1.2.1
+tld==0.11.11
+tls-parser==1.2.2
+token-bucket==0.3.0
+toml==0.10.2
+tomli==2.0.1
+torbrowser-launcher==0.3.5
+tornado==6.2
+tqdm==4.64.0
+traitlets==5.8.0
+translationstring==1.4
+tweepy==4.10.1
+Twisted==22.4.0
+txaio==21.2.1
+typeguard==2.13.3
+typing-extensions==3.10.0.2
+typing-inspect==0.7.1
 tzdata==2022.6
-django-phonenumber-field==7.0.0 # enables localization of phone number based on couuntry code
-Pillow==9.3.0
-drf-nested-routers
-model_bakery
-pytest-django
-django-filter==22.1
-phonenumbers==8.13.0
-black==22.12.0 # used for formatting code
\ No newline at end of file
+tzlocal==4.2
+u-msgpack-python==2.3.0
+ufoLib2==0.13.1
+ujson==5.3.0
+unicodecsv==0.14.1
+unicodedata2==14.0.0
+Unidecode==1.3.4
+unidiff==0.7.3
+uri-template==1.2.0
+urllib3==1.26.9
+urwid==2.1.2
+uvicorn==0.17.6
+uvloop==0.16.0
+venusian==3.0.0
+virtualenv==20.14.0+ds
+visions==0.7.5
+wafw00f==2.1.0
+wapiti3==3.0.4
+wcwidth==0.2.5
+webargs==8.0.1
+webcolors==1.12
+webencodings==0.5.1
+WebOb==1.8.6
+websocket-client==1.2.3
+websockets==10.2
+websockify==0.10.0
+Werkzeug==2.0.2
+wfuzz==3.1.0
+whois==0.8
+winacl==0.1.2
+wrapt==1.14.1
+wsaccel==0.6.3
+wsproto==1.1.0
+WTForms==2.2.1
+wxPython==4.0.7
+xcffib==0.11.1
+xdg==5
+xlrd==1.2.0
+XlsxWriter==3.0.2
+xlutils==2.0.0
+xlwt==1.3.0
+xmltodict==0.12.0
+yara-python==4.2.0
+yarl==1.7.2
+yaswfp==0.9.3
+zc.lockfile==2.0
+zipp==1.0.0
+zlib-wrapper==0.1.3
+zope.component==5.0.1
+zope.deprecation==4.4.0
+zope.event==4.4
+zope.hookable==5.1.0
+zope.interface==5.4.0
```

Comment:
You haven't used a vitual enviroment to run this project hence the enormous number of mudules coming from your pip freeze. 
The changes you made don' t warrant installing all these packages to our server machine. 
to mitigate this clone the project onto a new location the create a virtual environment. from there you activate the env and install dependancies.
This is the source of the error @jumalaw98 noted earlier.  also check on pr no #74 for further clarification